owner/src/main/java/org/the_chance/honeymart/ui/addCategory/AddCategoryListener.kt

```diff
@@ -0,0 +1,8 @@
+package org.the_chance.honeymart.ui.addCategory
+
+interface AddCategoryListener {
```

Comment:
Nice work using interaction listener 👏, you can just update the name to match our naming rules.
"AddCategoryInteractionListener"

Reply:
done